### PR TITLE
Feature - add /our-work/ page

### LINKED ISSUE #16: Build "Our Work" page

### PR DESCRIPTION
**Pull request by lukey-luke**
Fixes #16 
Figma here:
https://www.figma.com/design/rOuecpMAXScYt598h0Xiet/Verdance-Website-2.0?node-id=1145-1596&t=I5OAPK1N8t66Kv7L-0

### CODE REVIEW

**Comment by amprokop** [comment]
Requesting content updates!

We take on impactful, challenging projects.

Verdance tackles technology challenges that move the needle on making public services more effective. Check out some examples of projects we've worked on over the past few years.



Department of Veterans Affairs

FULLSTACK DEVELOPMENT & DATA SCIENCE

Getting well-earned benefits into the hands of Veterans

VA offers a wide variety of benefits to Veterans, but determining a Veteran's eligibility can be tricky and time-consuming. The PACT Act, passed in 2022, was a historic expansion of VA benefits, and required sweeping changes to handle new requirements and an increased volume of applications. We’ve worked on several efforts to improve the digital services that support Veterans, including redesigning benefits applications and creating new tools for VA employees to process claims more efficiently and accurately.





Centers for Medicare & Medicaid Services

API DESIGN & DEVELOPMENT

Removing blockers and creating new pathways for health data to be accessible and useful

CMS offers a variety of APIs to ensure Medicare providers can access the health data they need to give the most informed care possible to patients. We've helped build out health data APIs and the systems that support them, while ensuring that critical CMS systems are secure, scalable, and adaptable.




FRONTEND DEVELOPMENT

Centers for Disease Control and Prevention

Developing systems for pandemic readiness

The challenges of the early COVID years demonstrated a strong need for improved national technology infrastructure so healthcare jurisdictions can seamlessly and accurately report disease information. We've helped build a front door used by 35+ states to report disease data to help government shore up its capacity to respond to public health events.

**Comment by amprokop** [comment]
@lukey-luke also it doesn't appear you can navigate to this page from mobile currently, bc the "Our work" page doesn't have room in the header. Might need to confer with @Kate-allbee on the best approach - let's also keep in mind that we're going to want to have "CONTACT US" added next, and possibly "INSIGHTS" (for a company blog) after that.

File: src/_includes/our_work_org_card.liquid
Comment (etanb) [inline src/_includes/our_work_org_card.liquid:5]: we shouldn't be using inline styles.

File: src/_includes/our_work_org_card.liquid
Comment (etanb) [inline src/_includes/our_work_org_card.liquid:19]: if we need true utility classes, we should add them to `_foundation`, otherwise we should just make a class for this section within `org-card`, ie,

```
.org-card {
  &__content {}
}
```

or something semantically similar.

Comment (lukey-luke) [reply src/_includes/our_work_org_card.liquid]: This is pretty specific to `/our-work/` so I opted for the latter!

File: src/_includes/our_work_org_card.liquid
Comment (lukey-luke) [reply src/_includes/our_work_org_card.liquid]: moved style to 
```
    .org-logo {
      width: auto;
      max-height: 84px;

      &__mobile {
        margin-top: 120px;
      }
    }
```

File: src/assets/img/partners/CDC-logo.png
Comment (etanb) [inline src/assets/img/partners/CDC-logo.png:1]: nit: do we need this file now?

File: src/assets/img/screenshots/DPC.png
Comment (etanb) [inline src/assets/img/screenshots/DPC.png:1]: Sorry to be a nit, but can we get the original screenshot here and do the background with CSS? Looks like this color is `#BFD4BF` in hex.

File: src/assets/img/screenshots/Report Stream.png
Comment (etanb) [inline src/assets/img/screenshots/Report Stream.png:1]: Sorry to be a nit, but can we get the original screenshot here and do the background with CSS? Looks like this color is `#549371` in hex.

File: src/assets/img/screenshots/VA.png
Comment (etanb) [inline src/assets/img/screenshots/VA.png:1]: Sorry to be a nit, but can we get the original screenshot here and do the background with CSS? Looks like this color is `$dark-linen` in our color system.

**Comment by amprokop** [comment]
@lukey-luke - I'd like Nava to approve this before merging since they're prime, once we're ready, let's hold off until I confirm with them.

**Comment by amprokop** [comment]
Ok, Nava approved. We're good to go once this is ready.

File: src/assets/img/partners/CDC-logo.png
Comment (lukey-luke) [reply src/assets/img/partners/CDC-logo.png:1]: removed

File: src/assets/styles/_page.scss
Comment (amprokop) [inline src/assets/styles/_page.scss:234]: Can we bump this to 80px? It makes the page look a bit less crowded.

**Review by amprokop — APPROVED** [review]

**Pull request merged**
